src/Aspire.Hosting/SqlServer/SqlServerBuilderExtensions.cs

```diff
@@ -22,7 +22,7 @@ public static class SqlServerBuilderExtensions
     /// <returns>A reference to the <see cref="IResourceBuilder{SqlServerContainerResource}"/>.</returns>
     public static IResourceBuilder<SqlServerContainerResource> AddSqlServerContainer(this IDistributedApplicationBuilder builder, string name, string? password = null, int? port = null)
     {
-        password = password ?? Guid.NewGuid().ToString("N");
+        password = password ?? Guid.NewGuid().ToString("N") + Guid.NewGuid().ToString("N").ToUpper();
```

Comment:
This too short? Did you mean to do it twice?

Reply:
Yep. This is in lieu of a more elaborate mechanism.

Reply:
It was not too short, the password was not meeting the requirements of coming from the required 3 of 4 sets.

"The password must be at least 8 characters long and contain characters from three of the following four sets: Uppercase letters, Lowercase letters, Base 10 digits, and Symbols.."

By making it longer in this way, the password happens to have crossed the threshold for sets, i.e. Uppercase letters letters have been added (to lowercase letters and digits)

But more generally, this is not cryptographically secure.  Do we need an issue to track generating cryptographically secure secrets?